src/octopeer-github/content/ContentController.ts

```diff
@@ -50,21 +50,25 @@ class ContentController {
     }
 
     /**
-     * Hook the product of ElementBindings and ElementSelectors to the DOM-tree.
+     * Hook both the semantic and syntactic elements and events to the DOM.
      * @param database   the database that should be used when logging.
      */
     private hookToDOM(database: DatabaseAdaptable) {
+        this.hookSemanticToDOM(database);
+        this.hookSyntacticToDOM(database);
+    }
+
+    /**
+     * Hook the product of ElementBindings and ElementSelectors to the DOM-tree.
+     * @param database   the database that should be used when logging.
+     */
+    private hookSemanticToDOM(database: DatabaseAdaptable) {
         const esbFactory = new ElementSelectionBehaviourFactory();
         const eebFactory = new ElementEventBindingFactory();
         let elementEventBinding: ElementEventBindingData;
         let elementSelectionBinding: ElementSelectionBehaviourData;
         let elementEventBindingHolder: ElementEventBinding;
         let elementSelectionBindingHolder: ElementSelectionBehaviour;
```

Comment:
While you're at it, can you rename these variables? As they have different purposes now. Suggestion:
```
        let elementEventBindingData: ElementEventBindingData;
        let elementSelectionBindingData: ElementSelectionBehaviourData;
        let elementEventBinding: ElementEventBinding;
        let elementSelectionBehaviour: ElementSelectionBehaviour;
```